.github/ISSUE_TEMPLATE/help.md

```diff
@@ -5,42 +5,71 @@ labels: 'type: question'
 
 ---
 
-<!-- EssentialsX help guide
+<!-- EssentialsX help request guide
 
-Don't write inside the arrows as they will be hidden when you post your issue.
+NOTE: Failure to fill out this template properly may result in your issue being
+      delayed or ignored.
 
-Want faster support? Come join our Discord server to get help sooner:
-https://discord.gg/F7gexAQ
+Don't put anything inside this block, as it will be hidden when you post your
+issue. Make sure to also not type between any arrows in the template, as this
+text will also be hidden.
```

Comment:
Just a thought, but perhaps clarify what "this block" means, since I'd expect most of the people putting text inside it don't necessarily know what that means. Something like this, perhaps?
```suggestion
Don't type anything until the "Information" subsection below, as it will be
otherwise be hidden when you post your issue. Also make sure not to write
anything between any arrows in the template, as this text will also be hidden.
```